tools/dockerfile-llb-frontend/build/build.go

```diff
@@ -251,11 +253,16 @@ func execKraftkitBuild(base llb.State, config Configuration) llb.State {
 		Root()
 }
 
+// This creates the /unikraft/bin directory (corresponds to our WellKnownKernelPath)
+func createKernelDirectory(base llb.State, _ Configuration) llb.State {
+	return base.File(llb.Mkdir("/unikraft", 0o755)).
+		File(llb.Mkdir("/unikraft/bin", 0o755))
```

Comment:
```suggestion
	return base.File(llb.Mkdir(filepath.Dir(filepath.Dir(oci.WellKnownKernelPath)), 0o755)).
		File(llb.Mkdir(filepath.Dir(oci.WellKnownKernelPath), 0o755))
```

Let's use the wellknown path here too.

By the way, doesn't the `llb` have something similar to `MkdirAll`?

`return base.File(llb.MkdirAll(filepath.Dir(oci.WellKnownKernelPath), 0o755))`

Reply:
Thanks. There's WithParents(true).